database/redis/latest.Dockerfile

```
@@ -8,4 +8,7 @@ LABEL org.opencontainers.image.licenses=MIT
 
 # Update package index and install dependencies
 RUN apt-get update && \
-    apt-get install -y git curl iputils-ping telnet vim unzip
\ No newline at end of file
+    apt-get install -y git curl iputils-ping telnet vim unzip
+
+RUN apt-get update && apt-get install -y locales 
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)